src/stories/src/components/expansion-panel/expansion-panel.mdx

```diff
@@ -158,9 +160,10 @@ Forces the expansion panel to expand/collapse immediately **without** transition
 
 ## Accessibility
 
+- If the user is able to click the expansion panel to toggle its state, ensure that a `<button>` element is present and clearly labeled to serve as an accessible point of interaction.
+  - Set the `aria-expanded` atribute on the button to reflect the state of the component.
+  - Set `aria-controls` on the button to reference the id of the expandable content.
```

Comment:
Maybe we could elaborate on this a little bit above with an example code snippet of what to do?